app/src/ui/app.tsx

```diff
@@ -242,7 +242,7 @@ export class App extends React.Component<IAppProps, IAppState> {
         return this.push()
       case 'pull':
         return this.pull()
-      case 'create-commit':
+      case 'show-changes':
         return this.createCommit()
```

Comment:
@donokuda can we change the createCommit() methods to be called showChanges() also?

Reply:
oh good catch. Done in 12424a8